Application/Profiles/Profile.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Application.Profiles
+{
+    public class Profile
+    {
```

Comment:
Is there a difference between a profile and an AppUser?

Reply:
An AppUser can also host an event, while the Profile (now changed to AttendeeProfile) represents the people who participate.